src/Form/Element/PhoneNumber.php

```diff
@@ -96,6 +97,9 @@ public function getInputSpecification(): array
         return [
             'name'       => (string) $this->getName(),
             'required'   => true,
+            'filters'    => [
+                ['name' => StringTrim::class],
+            ],
```

Comment:
The validator won't let through anything that can't be parsed as a valid phone number. Invalid data is escaped on output so I don't think there's any need. Trimming is purely so that extra whitespace from user input is ignored rather than causing a validation failure.

Reply:
> Invalid data is escaped on output…

👍🏻